docs/odcs.rst

```diff
@@ -0,0 +1,120 @@
+ODCS Integration
+================
+
+When a container build is started, the user usually provides the URL to a yum
+repo that contains the packages to be installed when building the container
+image. Usually, this is a puddle containing signed packages.
+
+In the case a yum repo URL is not provided, OSBS falls back to the yum repo
+associated with the build tag for the Koji build target (required parameter for
```

Comment:
It's actually optional I think, but I'm not sure if that's because Koji integration is optional, i.e. maybe it's required when Koji integration is enabled, not sure.

Reply:
It depends from which POV. If you're kicking this off via rpkg, or koji-cli, it's a required parameter. Should I just remove the "required parameter ..." bit?

Reply:
Reworded this paragraph to indicate this applies when Koji integration is enabled.